packages/components/src/components/Alert/Alert.tsx

```diff
@@ -29,15 +23,15 @@ export type AlertProps = {
   /** Optional custom content to nest inside Alert
    * Use AlertContentColor or appropriate component props to set light/dark mode 'base' gray colors */
   children?: React.ReactNode
+  /** Optional boolean to determine whether the alert should be expandable */
+  expandable?: boolean
```

Comment:
Adding a comment below and thinking more on my general comment of the Jessica Q&A that only header-having Alerts  can be expandable, I wonder if that should be locked down with TS so `expandable` only becomes available as a prop after `header` is set.

Reply:
Added TS logic that requires a header for expandable Alerts